sdk/storage/azure-storage-queue/azure/storage/queue/_generated/aio/_azure_queue_storage.py

```diff
@@ -1,54 +1,51 @@
 # coding=utf-8
 # --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#
+# Licensed under the MIT License. See License.txt in the project root for license information.
 # Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
+from typing import Any
+
 from azure.core import AsyncPipelineClient
-from msrest import Serializer, Deserializer
+from msrest import Deserializer, Serializer
 
-from ._configuration_async import AzureQueueStorageConfiguration
-from azure.core.exceptions import map_error
-from .operations_async import ServiceOperations
-from .operations_async import QueueOperations
-from .operations_async import MessagesOperations
-from .operations_async import MessageIdOperations
+from ._configuration import AzureQueueStorageConfiguration
+from .operations import ServiceOperations
+from .operations import QueueOperations
+from .operations import MessagesOperations
+from .operations import MessageIdOperations
 from .. import models
 
 
 class AzureQueueStorage(object):
-    """AzureQueueStorage
-
-
-    :ivar service: Service operations
-    :vartype service: azure.storage.queue.aio.operations_async.ServiceOperations
-    :ivar queue: Queue operations
-    :vartype queue: azure.storage.queue.aio.operations_async.QueueOperations
-    :ivar messages: Messages operations
-    :vartype messages: azure.storage.queue.aio.operations_async.MessagesOperations
-    :ivar message_id: MessageId operations
-    :vartype message_id: azure.storage.queue.aio.operations_async.MessageIdOperations
+    """AzureQueueStorage.
 
-    :param url: The URL of the service account, queue or message that is the
-     targe of the desired operation.
+    :ivar service: ServiceOperations operations
+    :vartype service: azure.storage.queue.aio.operations.ServiceOperations
+    :ivar queue: QueueOperations operations
+    :vartype queue: azure.storage.queue.aio.operations.QueueOperations
+    :ivar messages: MessagesOperations operations
+    :vartype messages: azure.storage.queue.aio.operations.MessagesOperations
+    :ivar message_id: MessageIdOperations operations
```

Comment:
Are these intended to be ServiceOperations operation?

Reply:
it’s in generated not blocking

Reply:
That's a good question, I'll check to see if this is intended behavior